docs/content/node-operation/node-roles.mdx

```diff
@@ -49,14 +43,34 @@ Verification nodes are required to stake a minimum of 135,000 FLOW to be a confi
 
 ## Access
 
-Access nodes act as a proxy to the Flow network. The Access node routes transactions to the correct Collection node and routes state queries to available nodes (while likely caching state to answer queries in a timely manner in the future). Dapps and user agents can submit their transactions to any Access node or run their own if they can't find a service provider they're happy with.
+Access nodes act as a proxy to the Flow network. The Access node routes transactions to the correct collection node and routes state queries to execution nodes (while likely caching state to answer queries in a timely manner in the future). Clients submit their transactions to any Access node or run their own if they can't find a service provider they're happy with.
+
+Access nodes are required to stake a 100  FLOW to be a confirmed node operator. However, since an access node does not participate in block production, it does not receive any staking rewards.
```

Comment:
```suggestion
Access nodes are required to stake 100 FLOW to be a confirmed node operator. However, since an access node does not participate in block production, it does not receive any staking rewards.
```